lib/Remote/DesiredCapabilities.php

```diff
@@ -237,8 +237,9 @@ public function toW3cCompatibleArray()
             if (array_key_exists(ChromeOptions::CAPABILITY_W3C, $ossCapabilities)) {
                 $w3cCapabilities[ChromeOptions::CAPABILITY_W3C] = new \ArrayObject(
                     array_merge_recursive(
-                        (array) $ossCapabilities[ChromeOptions::CAPABILITY],
-                        (array) $ossCapabilities[ChromeOptions::CAPABILITY_W3C]
+                        // Encode and decode from JSON to cast the value to an array.
+                        (array) json_decode(json_encode($ossCapabilities[ChromeOptions::CAPABILITY])),
+                        (array) json_decode(json_encode($ossCapabilities[ChromeOptions::CAPABILITY_W3C]))
```

Comment:
Is this still necessary? The object is jsonserializable, so should be converted to array automatically by json_encode right before passing the value to curl.

Reply:
I’ll double-check but I think some of the unit tests were failing without this. I can roll this back and fix it in the unit tests instead if that’s the case. Will check shortly.

Reply:
Tests are passing without. This may have been required before because of the way that I was performing the changes prior to the updaet.